ai/ai21labs/v0/chat.go

```diff
@@ -0,0 +1,59 @@
+package ai21labs
+
+// Source https://docs.ai21.com/reference/jamba-instruct-api on 2024-07-21
+
+const chatEndpoint = "/studio/v1/chat/completions"
+
+type ChatMessage struct {
+	// enum: "user" "assistant" "system"
+	Role    string `json:"role"`
+	Content string `json:"content"`
+}
+
+type ChatRequest struct {
+	Model       string        `json:"model"`
+	Messages    []ChatMessage `json:"messages"`
+	MaxTokens   int           `json:"max_tokens"`
+	Temperature float32       `json:"temperature"`
+	TopP        float32       `json:"top_p"`
+	N           int           `json:"n"`
+	// not supported
+	// Stop   []string `json:"stop"` will trigger 500 internal server error
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500
If we have known the error, we should handle error.

Reply:
Fixed. Empty array of stop words will cause error from AI21labs